internal/internal_worker.go

```diff
@@ -559,6 +559,9 @@ type registry struct {
 	workflowVersioningBehaviorMap map[string]VersioningBehavior
 	activityFuncMap               map[string]activity
 	activityAliasMap              map[string]string
+	dynamicWorkflow               interface{}
+	dynamicOptions                DynamicRegisterOptions
```

Comment:
```suggestion
	dynamicWorkflowOptions                DynamicRegisterOptions
```
We may have activity options in the future